shub/image/list.py

```diff
@@ -52,31 +51,17 @@ def list_cmd_full(target, silent, version):
     image = config.get_image(target)
     version = version or config.get_version()
     image_name = utils.format_image_name(image, version)
```

Comment:
we need to fix this exception 

```
  File "/Users/alex/Dev/workspace/scrapinghub/repo/shub/shub/image/build.py", line 47, in build_cmd
    image_name = utils.format_image_name(image, version)
  File "/Users/alex/Dev/workspace/scrapinghub/repo/shub/shub/image/utils.py", line 96, in format_image_name
    parts = image_name.rsplit('/', 1)
AttributeError: 'NoneType' object has no attribute 'rsplit'
```

Happens when you don't have image config and use `shub image upload`